File: ChatServerVB/IClient.vb
Public Interface IClient
    Sub DisconnessionePartecipante(nome As String)
    Sub RiconnessionePartecipante(nome As String)
    Sub LoginPartecipante(client As User)
    Sub LogoutPartecipante(nome As String)
    Sub MsgTestoBroadcast(mittente As String, testo As String)
    Sub MsgImmagineBroadcast(mittente As String, ByVal img As Byte())
    Sub MsgTestoUnicast(mittente As String, testo As String)
    Sub MsgImmagineUnicast(mittente As String, ByVal img As Byte())
    Sub PartecipanteStaScrivendo(mittente As String)
End Interface

File: ChatServerCS/IClient.cs
namespace ChatServerCS
{
    public interface IClient
    {
        void DisconnessionePartecipante(string nome);
        void RiconnessionePartecipante(string nome);
        void AccessoPartecipante(Utente client);
        void LogoutPartecipante(string nome);
        void MsgTestoBroadcast(string mittente, string messaggio);
        void MsgImmagineBroadcast(string mittente, byte[] img);
        void MsgTestoUnicast(string mittente, string message);
        void MsgImmagineUnicast(string mittente, byte[] img);
        void PartecipanteStaScrivendo(string mittente);
    }
}